changes/ocd-3577.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE openchpl.job_type SET deleted = true WHERE name = 'Export Quarterly Report';
+UPDATE openchpl.job_type SET deleted = true WHERE name = 'Export Annual Report';
```

Comment:
If you need to update this for whatever reason, I'd encourage a new line at the end of the file so when we concat files together they have breaks in the final result. Not blocking approval though